web/core/src/jsTest/kotlin/elements/AttributesTests.kt

```diff
@@ -387,18 +434,31 @@ class AttributesTests {
             }
         }
 
-        assertEquals(
-            expected = "<button class=\"a\" style=\"color: red;\">Button</button>",
-            actual = nextChild().outerHTML
-        )
+        with(nextChild()) {
```

Comment:
the same question as in SVG test - I can not see why this is better (I apologise if I miss something)